tools/scoring/score/dimensions/standard_field_naming.py

```diff
@@ -37,25 +37,25 @@ def _split_subfields(self, field):
 
   def evaluate(self):
     """Calculates and assigns properties necessary for generating a score."""
-    proposed_translations, solution_translations = map(
-        self._condense_translations, (PROPOSED, SOLUTION))
-
     correct_subfields = []
     correct_ceiling: int = 0
     incorrect_subfields = []
 
-    for solution_field, solution_value in solution_translations:
-      solution_subfields = self._split_subfields(solution_field)
-      correct_ceiling += len(solution_subfields)
+    for entity_translations in self.translations.values():
+      for solution_field, solution_value in entity_translations[SOLUTION]:
+        solution_subfields = self._split_subfields(solution_field)
+        correct_ceiling += len(solution_subfields)
 
-      proposed_subfields = set()
+        proposed_subfields = set()
```

Comment:
indentation ok?

Reply:
Yes, everything got bumped over due to `:44` (which was, mercifully, the simple fix).